operator/cmd/mesh/operator-init.go

```diff
@@ -46,6 +46,9 @@ func addOperatorInitFlags(cmd *cobra.Command, args *operatorInitArgs) {
 	cmd.PersistentFlags().StringVar(&args.context, "context", "", ContextFlagHelpStr)
 	cmd.PersistentFlags().StringVar(&args.common.hub, "hub", hub, HubFlagHelpStr)
 	cmd.PersistentFlags().StringVar(&args.common.tag, "tag", tag, TagFlagHelpStr)
+	// nolint: lll
+	cmd.PersistentFlags().StringVar(&args.common.imagePullSecrets, "imagePullSecrets", "",
+		"The imagePullSecrets used to pull the operator image from private registry, could be secret list separated by comma, eg. 'imagePullSecret1,imagePullSecret2'")
```

Comment:
Better to create  `imagePullSecretsHelpStr` and use here.

Reply:
done, thanks